x/auth/client/testutil/suite.go

```diff
@@ -971,6 +971,39 @@ func (s *IntegrationTestSuite) TestCLIMultisignSortSignatures() {
 	s.Require().NoError(s.network.WaitForNextBlock())
 }
 
+func (s *IntegrationTestSuite) TestSignWithMultisig() {
+	val1 := s.network.Validators[0]
+
+	// Generate a account for signing.
+	account1, err := val1.ClientCtx.Keyring.Key("newAccount1")
+	s.Require().NoError(err)
+
+	addr1, err := account1.GetAddress()
+	s.Require().NoError(err)
+
+	// Generate multisig transaction.
```

Comment:
It's not a multisig below.

I recommend adding comments to explain what we're doing here, i.e only testing `--multisig` with an address not in keyring.